src/main/java/org/commcare/formplayer/session/FormplayerInstanceInitializer.java

```diff
@@ -91,15 +86,7 @@ protected InstanceRoot setupFixtureData(ExternalDataInstance instance) {
 
     @Override
     protected InstanceRoot setupSelectedCases(ExternalDataInstance instance) {
-        String guid = getGuidForSelectedCasesInstance(instance);
-        if (guid != null) {
-            VirtualDataInstance virtualDataInstance =
-                    virtualDataInstanceCache.read(UUID.fromString(guid));
-            if (virtualDataInstance != null) {
-                return new ConcreteInstanceRoot(virtualDataInstance.getRoot());
-            }
-        }
-        return ConcreteInstanceRoot.NULL;
+        return getExternalDataInstanceSource(instance, SessionFrame.STATE_MULTIPLE_DATUM_VAL);
```

Comment:
What do you think about putting this back into the parent class since it's now up to the remote instance fetcher to support?